controller_coverage.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Usage: bash ./controller_coverage.sh
+# Then run: firefox controller_coverage/index.html
+
+set -e
+set -x
+
+# delete coverage files
+#find .pio -name "*.gcno" -delete
+
+# run tests
+pio test -e native
+
+ENVIRONMENT=native
+OUTPUT_DIR=controller_coverage
+rm -rf $OUTPUT_DIR
+mkdir -p $OUTPUT_DIR
+lcov --directory .pio/build/$ENVIRONMENT --capture --output-file $OUTPUT_DIR/$ENVIRONMENT.info
```

Comment:
Perhaps this script should `cd "$(dirname "$0")"` at the beginning so that it can be run from any cwd?  As is, it looks like it will fail with an unhelpful error.  Note that variable quoting is important here so it still works if the path to this script contains a space.